handlers/messagebird/messagebird_test.go

```diff
@@ -0,0 +1,134 @@
+)
+
+var sigtestCases = []ChannelHandleTestCase{
+	{
+		Label:                "Receive Valid w Signature",
+		Headers:              map[string]string{"Content-Type": "application/json", "Messagebird-Signature-Jwt": validSignature},
+		URL:                  receiveURL,
+		Data:                 validReceive,
+		ExpectedRespStatus:   200,
+		ExpectedBodyContains: "Message Accepted",
+		ExpectedMsgText:      Sp("Test 3"),
+		ExpectedURN:          "tel:188885551515",
+		ExpectedDate:         time.Date(2023, time.July, 27, 00, 49, 29, 0, time.UTC),
+	},
+	{
+		Label:                "Bad Signature",
+		Headers:              map[string]string{"Content-Type": "application/json", "Messagebird-Signature-Jwt": invalidSignature},
+		URL:                  receiveURL,
+		Data:                 validReceive,
+		ExpectedRespStatus:   400,
+		ExpectedBodyContains: `{"message":"Error","data":[{"type":"error","error":"invalid request signature, signature expected: 28ce0ba19087fa7880ec08d2bab1c7d5f3e65c3b6ae093a0b261084671780339 got: 07ae5c6a96160aeb2e0dd8b0ea0561d3674b64a5a751b6e15c40452f561cb72e for body: '{\"receiver\":\"18005551515\",\"sender\":\"188885551515\",\"message\":\"Test again\",\"date\":1690386569,\"date_utc\":1690418969,\"reference\":\"1\",\"id\":\"b6aae1b5dfb2427a8f7ea6a717ba31a9\",\"message_id\":\"3b53c137369242138120d6b0b2122607\",\"recipient\":\"18005551515\",\"originator\":\"188885551515\",\"body\":\"Test 3\",\"createdDatetime\":\"2023-07-27T00:49:29+00:00\",\"mms\":false}'"}]}`,
+	},
+}
+
+var testCases = []ChannelHandleTestCase{
+	{
+		Label:                "Receive Valid w Sig",
+		Headers:              map[string]string{"Content-Type": "application/json", "Messagebird-Signature-Jwt": validSignature},
+		URL:                  receiveURL,
+		Data:                 validReceive,
+		ExpectedRespStatus:   200,
+		ExpectedBodyContains: "Message Accepted",
+		ExpectedMsgText:      Sp("Test 3"),
+		ExpectedURN:          "tel:188885551515",
+		ExpectedDate:         time.Date(2023, time.July, 27, 00, 49, 29, 0, time.UTC),
+	},
+	{
+		Label:                "Bad JSON",
+		Headers:              map[string]string{"Content-Type": "application/json", "Messagebird-Signature-Jwt": invalidSignature},
+		URL:                  receiveURL,
+		Data:                 "empty",
+		ExpectedRespStatus:   400,
+		ExpectedBodyContains: `{"message":"Error","data":[{"type":"error","error":"unable to parse request JSON: invalid character 'e' looking for beginning of value"}]}`,
+	},
+}
+
+func TestHandler(t *testing.T) {
+	RunChannelTestCases(t, testChannels, newHandler("MBD", "Messagebird", true), sigtestCases)
+	RunChannelTestCases(t, testChannels, newHandler("MBD", "Messagebird", false), testCases)
```

Comment:
I don't think we need the tests when we do not valid the signatures as we do not register a handler that way